src/stylesheets/components/_content-slider.scss

```diff
@@ -36,15 +44,18 @@
 .has-js {
   .slider-link {
     padding-right: $base-spacing-unit-large;
-    color: $links;
-    font-weight: bold;
-    text-decoration: underline;
-    cursor: pointer;
+    color: $headings;
+    //color: $links;
```

Comment:
Yes, because as soon as user testing is completed at the beginning of next week the content slider styling will go straight back in.